src/sidebar/oauth-auth.js

```diff
@@ -296,86 +255,12 @@ function auth($http, $rootScope, $window,
    * then exchange for access and refresh tokens.
    */
   function login() {
```

Comment:
The logic below for handling interaction with the OAuth popup window has moved into the `OAuthClient` class.